js-exercises/deep-copy-object/deepCopyObject.js

```diff
@@ -0,0 +1,24 @@
+const deepCopyObject = value => {
```

Comment:
You've declared 2 functions deepCopyObject & deepCopyOfObject. Names are very confusing. Please use some appropriate names.

Reply:
If you look closely deepCopyObject was already declared name in problem, Ideally, it should have been better. Keeping the name deepCopyOfObject was to define the purpose of that function that just iterates over the object.

Reply:
I think you can also rename what is provided.